src/components/SelectComponent/SelectComponent.scss

```diff
@@ -0,0 +1,37 @@
+@use "../../styles/_reset.scss" as *;
+@use "../../styles/_variables.scss" as *;
+
+.select {
+    border: solid 1px black;
+    border-radius: 0.5rem;
+    width: 32rem;
+    padding-left: 0.5rem;
+    height: 3rem;
+    display: flex;
+    align-items: center;
+    
```

Comment:
Add a background colour as it is transparent please and drop down text is too light. :)